orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -1117,10 +1119,8 @@ void RetrieveSendRecvOperators(
   }
 }
 
-PathString GenerateFileNameWithIndex(const PathString& base_str, int index, const PathString& file_suffix) {
-  Path p;
-  ORT_ENFORCE(Path::Parse(base_str, p).IsOK());
-  return p.ConcatIndex(index).Concat(file_suffix).ToPathString();
+PathString GenerateFileNameWithIndex(const std::string& base_str, int index, const std::string& file_suffix) {
+  return path_utils::MakePathString(base_str, index, file_suffix);
```

Comment:
since MakePathString is used instead, Path::Concat() is no longer used, right? do we still need it?

Reply:
Let's keep it since it is already been implemented. It is a general function and could be used in the future.

Reply:
hm, i'd prefer to not have unused code. any number of things could be useful in the future :)
but that's ok if you want to leave it.